documentation/book/proc-upgrading-the-cluster-operator.adoc

```diff
@@ -0,0 +1,70 @@
+// This module is included in the following assemblies:
+//
+// assembly-upgrade.adoc
+
+[id='proc-upgrading-the-cluster-operator-{context}']
+= Upgrading the Cluster Operator from 0.9.0 to 0.10.0
+
+This procedure will describe how to upgrade a Cluster Operator deployment from version 0.9.x to version 0.10.0. 
+The availability of Kafka clusters managed by the Cluster Operator is not affected by the upgrade operation.
+
+== Prerequisites 
+
+* An existing version 0.9.0 Cluster Operator deployment to be upgraded.
```

Comment:
Is this not relevant for any other version?

Reply:
No.